src/config_test.js

```diff
@@ -39,6 +39,10 @@ module.exports = {
         url = config.moduleUrl("ace/ext/textarea");
         assert.equal(url, "a/b/ext-textarea.js");
         config.set("packaged", false);
+        
+        var callback = () => "testCallback";
+        config.setModuleLoader("ace/test-module", callback);
+        assert.equal(config.dynamicModules["ace/test-module"], callback);
```

Comment:
Could you expand this test to also call loadModule and check the result?

Reply:
Done!